backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Fresh_PriceList_Details_ToDate_Report.sql

```diff
@@ -68,17 +76,32 @@ BEGIN
     CREATE TEMP TABLE IF NOT EXISTS temp_price_list_details AS
     SELECT DISTINCT bp.c_bpartner_id,
                     plv.M_PriceList_Version_ID,
-                    plv.name
+                    plv.name,
+                    (SELECT bpl.c_bpartner_location_id
+                     FROM c_bpartner_location bpl
+                     WHERE bpl.c_bpartner_id = bp.c_bpartner_id
+                     ORDER BY bpl.isbilltodefault
+                             DESC
+                     LIMIT 1) AS c_bpartner_location_id,
+                    bp.ad_org_id
```

Comment:
Why taking the ID, when you need to display the address content.

Reply:
to passed as parameter to `de/metas/docs/generics/report_bpblock_shorter_left_margin.jasper` sub report